css/style.css

```diff
@@ -1,9 +1,379 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose-compressed-trial');
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,300;1,400&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
 body {
-  background-color: black;
+  background-color: #d3d3d3;
+  font-family: 'Roboto', 'cocogoose', sans-serif;
+  color: #272a31;
+}
+
+.project-logo {
+  width: 76px;
+  height: 64px;
+  border-radius: 76px;
+  margin-left: 72px;
+}
+
+.nav-bar {
+  width: 100%;
+  flex-direction: column;
+  color: #555;
+  background-color: #fff;
+}
+
+.navigation-link {
+  width: 100%;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.f-direction-col {
+  flex-direction: column;
+}
+
+.display-none {
+  display: none;
+}
+
+.f-direction-row {
+  flex-direction: row;
+}
+
+.f-wrap {
+  flex-wrap: wrap;
+}
+
+.j-content-btw {
+  justify-content: space-between;
+}
+
+.j-content-cnt {
+  justify-content: center;
+}
+
+.text-align-center {
+  text-align: center;
+}
+
+.align-items {
+  align-items: center;
+}
+
+.toggle-button {
+  position: absolute;
+  right: 0.72rem;
+  top: 0.4rem;
+  z-index: 2;
+  padding: 8px 16px;
+  background-color: #fff;
+}
+
+.x-button {
+  position: absolute;
+  right: 0.72rem;
+  top: 0.4rem;
+  z-index: 3;
+  background-color: #fff;
+  font-size: 20px;
+  font-weight: bolder;
+  margin-right: 24px;
+}
+
+.bar {
+  transition: 0.4s;
+  background-color: #272a31;
+  display: block;
+  height: 4px;
+  left: 28px;
+  top: 50%;
+  width: 25px;
+}
+
+.bar:first-child {
+  transform: translateY(-4px);
+}
+
+.bar:last-child {
+  transform: translateY(4px);
+}
+
+.nav-ul {
+  flex-direction: column;
+  width: 100%;
+  padding: 16px;
+  position: fixed;
+  z-index: 1;
+  background-color: #fff;
+  top: 0;
+  height: 100vh;
+}
+
+.nav-ul-li {
+  margin: 54px 0;
+  font-size: 17px;
+}
+
+.nav-li-anchor {
+  color: #272a31;
+}
+
+.cc-campaign {
+  border: 2px solid #ec5242;
+  padding: 3px 6px;
+}
+
+.nav-li-anchor:hover {
+  background-color: #000;
+  padding: 12px 17px;
+  color: #fff;
+}
+
+.home-page {
+  /* background-image: url(../images/refugee.jpg); */
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.home-page-first-div {
+  padding: 14px;
+  margin-bottom: 95px;
+}

```

Comment:
- [ ] Kindly as asked by the previous reviewer, Your first section doesn't look like the Original design. You can just make the `padding` bigger ex `padding: 140px;`.

Your design
![image](https://user-images.githubusercontent.com/35535130/157995057-de8a363b-0dda-4862-b72a-b35558d445e2.png)

Original design 
![image](https://user-images.githubusercontent.com/35535130/157995090-b4a1612e-1bea-495b-b84e-03d313141284.png)

Reply:
Thanks for the review changes made